magento/magento2
# MC-37807: Link to a file of a Product with Customizable Option(File) is not available throughout a multishipping checkout process

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#

### Manual testing scenarios (*)

Steps to Reproduce:

1. Go to the Storefront as the created Customer
2. Add the created Product to the Shopping Cart (Qty = 2), uploading a valid file
3. Open the mini-cart
4. Click the “View and Edit Cart” link
5. Click the “Check Out with Multiple Addresses” link
6. Select shipping addresses on the “Ship to Multiple Addresses”
7. Click the “Go to Shipping Information” button
8. Select Flat Rate - Fixed Shipping Method for the each address
9. Click the “Continue to Billing Information” button
10. Select “Check / Money order” Payment Method
11. Click the “Go to Review Your Order” button

**Expected results:**
There should be a link, enabling file downloading, throughout entire multishipping checkout process (name of the file)

 **Actual results:**
There is no link enabling file downloading throughout entire multishipping checkout process (name of the file)


### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#31095: MC-37807: Link to a file of a Product with Customizable Option(File) is not available throughout a multishipping checkout process
